include/handlers.hpp

```diff
@@ -65,9 +66,21 @@ class BillingHandler:
     }
     else
     {
-      return &HttpStack::DEFAULT_SAS_LOGGER;
+      if (_http_acr_logging)
+      {
+        // Include bodies in ACR HTTP messages logged to SAS.
+        return &HttpStack::DEFAULT_SAS_LOGGER;
+      }
+      else
+      {
+        // Ommit bodies from ACR HTTP messages logged to SAS.
+        return &HttpStack::PRIVATE_SAS_LOGGER;
```

Comment:
I'll add a comment to explain our use of it.

FYI - the header comment for this logger in `httpstack.h` is:
```cpp
  /// SAS logger which omits bodies of requests and responses
  /// in SAS logs.
  class PrivateSasLogger : public DefaultSasLogger
```